LICENSE

```diff
@@ -0,0 +1,21 @@
+MIT License
```

Comment:
The dylan domain code was written by Dustin Voss and he put this in the files:

```
Copyright (c) 2011-2016 Open Dylan Maintainers. All rights reserved.
```

I am not a lawyer, but as both a "Dylan Hacker" and an "Open Dylan Maintainer" I feel like it's reasonable to change those to "Dylan Hackers" to match what we use elsewhere.